pkg/opa/opa_calls.go

```diff
@@ -38,7 +39,7 @@ const opaInstanceURLFormat = opaCheckInstancesURLFormat + "/%s"
 // GetChecks queries Fairwinds Insights to retrieve all of the Checks for an organization
 func GetChecks(org, token, hostName string) ([]opaPlugin.OPACustomCheck, error) {
 	url := fmt.Sprintf(opaURLFormat, hostName, org)
-	logrus.Infof("OPA URL: %s", url)
+	logrus.Debugf("OPA URL: %s", url)
```

Comment:
any reason for changing this from `infof`?

Reply:
The thought in changing the log level here is that a user will not want to see the Insights URL for typical usage, however if they want to verify the URL they can enable debug logging (with the `--log-level debug` flag). Changing this from info to debug  helps to clean up the output.